pages/glossary/foreground-colors-of-element.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Foreground Colors Of Element
+key: foreground-colors-of-element
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The _foreground colors of an element_ are the [foreground colors](#foreground-colors-of-text) of all [visible](#visible) [text nodes](https://dom.spec.whatwg.org/#text) that are [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) of that element. 
```

Comment:
Not needed due to the "text nodes" precision.

The node itself is here an element, not a text node. So it will never be a "text node that is inclusive descendant". So there is no difference between "text nodes that are inclusive descendants" and "text nodes that are descendants".